src/QirRuntime/.clang-tidy

```diff
@@ -26,4 +26,3 @@ CheckOptions:
     value:  'CamelCase'
   - key:    readability-identifier-naming.NamespaceCase
     value:  'CamelCase'
-

```

Comment:
Nit: if you are not making intentional changes to the files, maybe, revert them, so no changes at all?

Reply:
Reverted the changes to .clang-tidy.